zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -209,7 +209,9 @@ def render_events(self, events_dict: Dict[str, Any]) -> List[str]:
             # Internal hyperlink name
             h3_id: str = event_type_str
             event_name: str = event_type_str
-            event_type_str = f'<span class="api-argument-required"> {event_type_str}</span>'
+            event_type_str = (
+                f'<span class="api-argument-required api-event-name"> {event_type_str}</span>'
+            )
```

Comment:
I think you should just drop the `api-argument-required` class and just have an `api-event-name` class with whatever styling it requires and no `!important`?